### PR TITLE
Add changelog for #15633

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

File: doc/changelog/09-coqide/15633-change-15633.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+- **Fixed:**
+  anomaly when using proof diffs with no focused goal
+  (`#15633 <https://github.com/coq/coq/pull/15633>`_,
+  fixes `#15578 <https://github.com/coq/coq/issues/15578>`_,
+  by Gaëtan Gilbert).
```
Comment (jfehrle) [inline doc/changelog/09-coqide/15633-change-15633.rst:5]:
```suggestion
  by Jim Fehrle).
```

**Review by jfehrle — APPROVED** [review]

**Comment by jfehrle** [comment]
@coqbot: merge now

**Comment by jfehrle** [comment]
@coqbot: merge now

Alas, it's not possible to assign or add labels from my phone :-(

**Comment by jfehrle** [comment]
@coqbot: merge now

**Comment by Zimmi48** [comment]
> Alas, it's not possible to assign or add labels from my phone :-(

My GitHub Android application supports this.

**Comment by Alizter** [comment]
Same here, though it is definitely not obvious. When browsing a PR or issue there is an `(i)` in the bottom right corner. From there you can edit the assignees and labels etc.

**Comment by Alizter** [comment]
From the mobile browser version, if you scroll to the very bottom of the PR you can see the labels.

**Comment by jfehrle** [comment]
👍 (don't see how I can add an emoji to your last comment in the browser)

**Pull request merged**